server/routerlicious/packages/routerlicious-base/src/alfred/routes/api/documents.ts

```diff
@@ -52,6 +52,75 @@ import { Constants, getSession, StageTrace } from "../../../utils";
 import { IDocumentDeleteService } from "../../services";
 import type { RequestHandler } from "express-serve-static-core";
 
+/**
+ * Response body shape for modern clients that can handle object responses.
+ * @internal
+ */
+interface ICreateDocumentResponseBody {
+	/**
+	 * The id of the created document.
+	 */
+	id: string;
```

Comment:
Should these properties be `readonly`?

Reply:
TIL that you can use `readonly` in typescript interfaces, not just classes! Thanks!